raiden/api/python.py

```diff
@@ -461,25 +481,25 @@ def set_total_channel_deposit(
             channel_id=channel_state.identifier,
         )
 
-        balance = token.balance_of(self.raiden.address)
+        total_network_balance = token.balance_of(registry_address)
 
-        if self.raiden.config['environment_type'] == Environment.PRODUCTION:
-            deposit_limit = (
-                token_network_proxy.proxy.contract.functions.
-                channel_participant_deposit_limit().call()
-            )
-        elif self.raiden.config['environment_type'] == Environment.DEVELOPMENT:
-            deposit_limit = (
-                token_network_proxy.proxy.contract.functions.
-                deposit_limit().call()
+        if total_network_balance + total_deposit > per_token_network_deposit_limit:
```

Comment:
The calculation here is wrong. It should be `total_network_balance + addendum` where `addendum` is defined [here](https://github.com/raiden-network/raiden/pull/3581/files#diff-8d55798a5e7b4afffecee7c4dbf6f86eR508) a few lines down. 

That's because if this is a secondary call to total deposit and we are just increasing our total deposit, we may think we hit the limit with the current code where in fact we did not